index.html

```diff
@@ -11,32 +11,201 @@
 			body {
 				font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Helvetica Neue", Arial, sans-serif;
 			}
+
+			#search-field-label {
+				display: block;
+			}
+
+			.container {
+				max-width: 1028px;
+				margin: 0 auto;
+			}
+
+			.page-title, .search-form {
+				text-align: center;
+			}
+
+			.article-image {
+				height: 127px;
+				width: 190px;
+				flex: 1;
+			}
+
+			.text-container {
+				flex: 3;
+				margin-left: 1em;
+			}
+
+			.result-item {
+				display: flex;
+				margin: 2em;
+				padding: 1em;
+				border: 2px solid rgba(0, 0, 0, .5);
+			}
+
+			#search-field, #search-button {
+				height: 2em;
+				font-size: 18px;
+			}
+
+			#search-button {
+				margin-left: 1em;
+				display: inline-block;
+			}
+
 		</style>
 	</head>
 
 	<body>
-		<h1>Thrilling Frontend Technical Challenge</h1>
+		<div class="container">
+			<h1 class="page-title">Thrilling Frontend Technical Challenge</h1>
+
+			<form class="search-form">
+				<input type="text" placeholder="Enter Search Term" id="search-field" />
+				<input type="button" value="Search" id="search-button" />
```

Comment:
consider changing input type to `submit` or replacing with
```suggestion
				<button type="submit>Search</button>
```
See comment on line 77 below for further explanation.